packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -494,11 +494,11 @@ SwipeableDrawer.defaultProps = {
   disableDiscovery: false,
   disableSwipeToOpen:
     typeof navigator !== 'undefined' && /iPad|iPhone|iPod/.test(navigator.userAgent),
+  hysteresis: 0.6,
   swipeAreaWidth: 20,
-  hysteresis: 0.5,
-  velocity: Infinity,
   transitionDuration: { enter: duration.enteringScreen, exit: duration.leavingScreen },
   variant: 'temporary', // Mobile first.
+  velocity: 1,
```

Comment:
is the default velocity value a breaking change?
And shouldn't we set the value to 0.4 like google does (as @leMaik mentioned)?